.gitignore

```diff
@@ -9,3 +9,4 @@ Thumbs.db
 *.so
 coverage.*
 
+/example/metric/metric
```

Comment:
What is that file btw?

Reply:
It's a binary generated during the build I think. `make precommit` complained about this file appearing and being uncommitted.